### ISSUE TITLE
"virtual" keyword not ignored for :Implement

### ISSUE THREAD

**Issue by Shatur** [d86leader/vim-cpp-helper#10]
For example the following deginition:
```cpp
virtual void functionExample();
```
will have the following implementation:
```cpp
virtual void ClassName::functionExample()
{
}
```
but expected without `virtual` keyword:
```cpp
void ClassName::functionExample()
{
}
```

**Comment by maurges** [comment]
That was a small oversight. Fixed: 386beffb16ea981dd7cd83c7efbfc8726b0a6754

**Comment by Shatur** [comment]
Thanks!
